### PR TITLE
[CAT-2064] Editing text field in formula editor should be larger than one line

### PR DESCRIPTION
**Pull request by singhalok641**
Please review this.

### CODE REVIEW

File: .idea/.name [hunk 1]
```diff
@@ -1 +0,0 @@
-Catroid
```
Comment (thomasmauerhofer) [inline .idea/.name]: Please exclude file like @thmq sayed at https://github.com/Catrobat/Catroid/pull/2075

File: .idea/codeStyleSettings.xml [hunk 2]
```diff
@@ -1,266 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (thomasmauerhofer) [inline .idea/codeStyleSettings.xml]: Please exclude file like @thmq sayed at https://github.com/Catrobat/Catroid/pull/2075

File: .idea/.name (hunk 1 above)
Comment (thomasmauerhofer) [inline .idea/.name]: Please exclude file like @thmq sayed at https://github.com/Catrobat/Catroid/pull/2075

File: .idea/copyright/Catroid.xml [hunk 3]
```diff
@@ -1,9 +0,0 @@
-<component name="CopyrightManager">
```
Comment (thomasmauerhofer) [inline .idea/copyright/Catroid.xml]: Please exclude file like @thmq sayed at https://github.com/Catrobat/Catroid/pull/2075

File: .idea/copyright/profiles_settings.xml [hunk 4]
```diff
@@ -1,3 +1,3 @@
 <component name="CopyrightManager">
-  <settings default="Catroid" />
```
Comment (thomasmauerhofer) [inline .idea/copyright/profiles_settings.xml]: Please exclude file like @thmq sayed at https://github.com/Catrobat/Catroid/pull/2075

File: .idea/encodings.xml [hunk 5]
```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (thomasmauerhofer) [inline .idea/encodings.xml]: Please exclude file like @thmq sayed at https://github.com/Catrobat/Catroid/pull/2075

File: .idea/misc.xml [hunk 6]
```diff
@@ -3,6 +3,30 @@
   <component name="EntryPointsManager">
```
Comment (thomasmauerhofer) [inline .idea/misc.xml]: Please exclude file like @thmq sayed at https://github.com/Catrobat/Catroid/pull/2075

File: .idea/vcs.xml [hunk 7]
```diff
@@ -1,18 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (thomasmauerhofer) [inline .idea/vcs.xml]: Please exclude file like @thmq sayed at https://github.com/Catrobat/Catroid/pull/2075

File: build.gradle [hunk 8]
```diff
@@ -8,7 +8,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.3'
+        classpath 'com.android.tools.build:gradle:2.2.3'
```
Comment (thomasmauerhofer) [inline build.gradle]: Why you have updated the gradle version?

File: Catroid.iml [hunk 9]
```diff
@@ -1,4 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module external.linked.project.path="$MODULE_DIR$" external.root.project.path="$MODULE_DIR$" external.system.id="GRADLE" external.system.module.group="" external.system.module.version="unspecified" type="JAVA_MODULE" version="4">
-</module>
-
+<module external.linked.project.id="Catroid" external.linked.project.path="$MODULE_DIR$" external.root.project.path="$MODULE_DIR$" external.system.id="GRADLE" external.system.module.group="" external.system.module.version="unspecified" type="JAVA_MODULE" version="4">
```
Comment (thomasmauerhofer) [inline Catroid.iml]: Please exclude file like @thmq sayed at https://github.com/Catrobat/Catroid/pull/2075

File: catroid/build.gradle [hunk 10]
```diff
@@ -26,7 +26,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.3'
+        classpath 'com.android.tools.build:gradle:2.2.3'
```
Comment (thomasmauerhofer) [inline catroid/build.gradle]: Why you have updated the gradle version?

Comment (singhalok641) [inline catroid/build.gradle]: I am using the latest version of Android Studio and it uses Instant Run feature which requires minimum gradle version 2.2.3.

File: Catroid.iml (hunk 9 above)
Comment (singhalok641) [inline Catroid.iml]: As @thmq  said I have run the gradle task:git./gradlew gitSkipWorktreeForIntellijConfigFiles before doing any commits which should ignore the files need to be ignored. Please guide me how do I exclude these files?

File: catroid/build.gradle (hunk 10 above)
Comment (thomasmauerhofer) [inline catroid/build.gradle]: Put please don't commit these change. There can be problems with jenkins, and so on. The gradle version will only be updated by the coordinators.

Comment (singhalok641) [inline catroid/build.gradle]: yeah, I'll change that, but what about the issue of excluding files. I have run the task as told by @thmq .

**Comment by singhalok641** [comment]
Can anyone please help me by explaining what needs to be done and a list of the files that need to be excluded before the pull request.
Thanks,

**Pull request closed without merging**